src/client/Microsoft.Identity.Client/Platforms/Features/WinFormsLegacyWebUi/WebBrowserBeforeNavigateEventArgs.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+
+namespace Microsoft.Identity.Client.Platforms.Features.WinFormsLegacyWebUi
+{
+    /// <summary>
+    /// Event arguments for <c>BeforeNavigate</c> event
+    /// </summary>
+    public class WebBrowserBeforeNavigateEventArgs : CancelEventArgs
+    {
+        /// <summary>
+        /// </summary>
+        public WebBrowserBeforeNavigateEventArgs(string url, byte[] postData, string headers, int flags,
+            string targetFrameName, object webBrowserActiveXInstance)
```

Comment:
nit: move all the parameters to a new line

Reply:
#Resolved